src/CarbonAware.CLI/test/integrationTests/Commands/EmissionsCommandTests.cs

```diff
@@ -83,4 +83,53 @@ public async Task Emissions_StartAndEndOptions_ReturnsExpectedData()
         Assert.IsNotNull(firstResult["Rating"]);
         Assert.IsNotNull(firstResult["Duration"]);
     }
+
+    [Test]
+    public async Task Emissions_BestOption_ReturnsExpectedData()
+    {
+        // Arrange
+        var start = DateTimeOffset.Parse("2022-09-01T00:00:00Z");
+        var end = DateTimeOffset.Parse("2022-09-01T03:00:00Z");
+        var location = "eastus";
+        _dataSourceMocker.SetupDataMock(start, end, location);
+
+        // Act
+        var exitCode = await InvokeCliAsync($"emissions -l {location} -s 2022-09-01T02:01:00Z -e 2022-09-01T02:04:00Z -b");
```

Comment:
This test is good as written, but could be even stronger if we setup and tested `emissions -l {locationOne} -l {locationTwo} -s 2022-09-01T02:01:00Z -e 2022-09-01T02:04:00Z -b` which would also test that it picks the single best from multiple locations.